src/text-buffer.coffee

```diff
@@ -1084,43 +1084,77 @@ class TextBuffer
   # {::backwardsScan} to avoid tripping over your own changes.
   #
   # * `regex` A {RegExp} to search for.
+  # * `options` (optional) {Object}
+  #   * `lineCountBefore` {Number} default `0`; The number of lines before the
+  #      matched line to include in the results object.
+  #   * `lineCountAfter` {Number} default `0`; The number of lines after the
+  #      matched line to include in the results object.
```

Comment:
After talking with @nathansobo about this new API, we decided that the following names would tell the story better:

* `lineCountBefore` -> `leadingContextLineCount`
* `lineCountAfter` -> `trailingContextLineCount`
* `linesBefore` -> `leadingContextLines`
* `linesAfter` -> `trailingContextLines`

Could you make those changes to the field names? I'll update `scandal` to match those names.